src/options.ts

```diff
@@ -81,6 +81,19 @@ export const getSourcemaps = (): string[] => {
   return sourcemapsOption.split(' ');
 };
 
+/**
+ * Dist is optional, but should be a string when provided.
+ * @returns string
+ */
+export const getDist = (): string | undefined => {
```

Comment:
```suggestion
export const getDist = (): string | null => {
```

It appears that all other options use `null` not `undefined`. Will change to null.